components/automate-deployment/pkg/assets/binds.go

```diff
@@ -13,6 +13,7 @@ automate-builder-api BINDING_MODE strict
 automate-builder-api-proxy REQUIRED automate-builder-api
 automate-builder-api-proxy BINDING_MODE strict
 automate-builder-memcached BINDING_MODE strict
+automate-cds REQUIRED compliance-service
```

Comment:
It is so easy to add a service to another service now. All I had to do is add the binding to the plan.sh file and run `deployment_generate`. 

Good job deployment team!